src/main/webapp/search-meal-style.css

```diff
@@ -0,0 +1,65 @@
+/* Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License. */
+
+body {
+  background-image: url("image/background-main.jpg");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: right top;
+  background-attachment: fixed;
+  font-family: sans-serif;
+}
+
+input[type=text], select {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+input[type=submit] {
+  width: 100%;
+  background-color: rgb(136, 235, 140);
+  color: white;
+  padding: 14px 20px;
+  margin: 8px 0;
+  border: none;
+  border-radius: 4px;
+}
+
+div {
```

Comment:
This selector will apply a certain width to all `div`s.

As mentioned above, type selectors are very broad and should only contain very general properties. Given than `div` is the most often used / nested selector, setting div for all of them will cause issues once we add more divs to our frontend.

I'd suggest not using a `div` selector here, but instead some id selector or class selector. I can see there's two divs in the index.html file. It's a better idea to wrap them with yet another `div`, give that `div` an id / class, and modify this selector to apply only to that specific class / id.

e.g.
in index.html
```
<div id=search-meal-block>
  <div>
    // form
  </div>
  <div>
     // dishes container
  </div>
</div>
```
in search-meal-style.css
```
#search-meal-block {
  // the css props
}
```

In short, if a few divs are expected to have the same size, it's usually better to wrap them with yet another div and set size on it.